test cases/common/16 configure file/meson.build

```diff
@@ -23,7 +23,10 @@ cfile)
 test('inctest', e)
 
 # Now generate a header file with an external script.
-genprog = find_program('python3')
```

Comment:
Should probably use the python3 module here since we don't use `find_program` on a script.